src/lib/falcor/FalcorController.js

```diff
@@ -1,15 +1,15 @@
 import _ from "lodash";
 import { isAppReady, expandCache, pathSetsInCache, validateFalcorPathSets } from "lib/falcor/falcorUtils";
 import BaseComponent from "lib/BaseComponent";
-import { setLoading, signalLeaving } from "lib/loader"
-import { uuid } from 'lib/utilities'
+import { setLoading, signalLeaving } from "lib/loader";
+import { uuid, setGlobalError, resetGlobalError, getGlobalError } from 'lib/utilities';
 
 // Abstract class for fetching falcor objects
 export default class FalcorController extends BaseComponent {
   constructor(props) {
     super(props)
     if (this.constructor == FalcorController) {
-      throw new TypeError("FalcorController is abstract")
+      setGlobalError(new TypeError("FalcorController is abstract"));
```

Comment:
This is a developer error, I don't think we would ever want to render an error message for this.